docs/getting-started.md

```diff
@@ -1,21 +1,162 @@
+8. [Close Zulip Terminal](#Close-Zulip-Terminal)
+
+*Note: You can always use <kbd>?</kbd> to open **help** and view keyboard shortcuts.*
+
+# Open the App
+
+Once you have [installed](https://github.com/zulip/zulip-terminal#installation) the app, use `zulip-term` to run it. You will be prompted to add the URL of organization you're a part of - use chat.zulip.org to join the Zulip community. If you have issues, check out our [Running for the First Time](https://github.com/zulip/zulip-terminal#installation) section.
+
+# Understand the Layout
 ![Screenshot](https://user-images.githubusercontent.com/9568999/106061037-b659a580-60a9-11eb-8ff8-ea1c54ac9084.png)
 
-Zulip Terminal is divided into three columns/divisions.
-**The left column** shows list of all the streams you are subscribed to and some buttons for viewing different types of messages.
-**The middle column** displays all the messages in the current narrow.
-**The right column** shows all the active(green)/idle(yellow)/offline(white) users.
+Zulip Terminal is divided into three columns/divisions:
++ **The left column** shows list of all the streams you are subscribed to and some keyboard shortcuts for viewing different types of messages.
++ **The middle column** displays all the messages in the current narrow.
++ **The right column** shows all the active(green)/idle(yellow)/offline(white) users.
+
+# Your Cursor
+
+When you first open Zulip Terminal, your cursor will be located in the upper left-hand corner on **All Messages**. You can tell because the section will be highlighted (see image below). You can use your cursor to navigate around Zulip Terminal - let's try moving the cursor around and see what happens. Try moving it left (<kbd>left</kbd>/<kbd>h</kbd>), right (<kbd>right</kbd>/<kbd>l</kbd>), up (<kbd>Up</kbd>/<kbd>k</kbd>) and down (<kbd>Down</kbd>/<kbd>j</kbd>).
+
+![Screenshot Showing Your Cursor](getting-started-imgs/your-cursor.png)
+
+# Keyboard Shortcuts
+
+Sometimes it can to take forever to manually move our cursor where we want it - that's why Zulip Terminal has keyboard shortcuts! Type <kbd>?</kbd> to open **help** and view the keyboard shortcuts that are available to you at your cursor's current position. **Help** (<kbd>?</kbd>) updates automatically as you move your cursor around (some shortcuts aren't available in certain sections). You can also view a list of [keyboard shortcuts here.](https://github.com/zulip/zulip-terminal#hot-keys)
+
+![Screenshot Showing Help](getting-started-imgs/help.png)
+
+Let's try out a few of these shortcuts out!
+
++ First, type <kbd>P</kbd> to view your Private Messages (PMs).
+
++ Second, type <kbd>#</kbd> to view all messages you were mentioned (@'ed) in.
+
++ Third, type <kbd>/</kbd> to jump to the search bar at the top of the in the middle column. Type <kbd>esc</kbd> to move your cursor out of the search box so we can use our keyboard shortcuts again (otherwise you'd just be typing them in the search box!).
```

Comment:
Accidentally missed a word: `search bar at the top of the` screen `in the middle column`